express_relay/sdk/js/src/examples/simpleSearcherEvm.ts

```diff
@@ -46,6 +46,8 @@ class SimpleSearcherEvm {
   }
 
   async opportunityHandler(opportunity: Opportunity) {
+    if (!("targetContract" in opportunity))
```

Comment:
Are we trying to validate fields and type here or just checking if it's a Svm or Evm opp?

Reply:
just checking it's an Svm/Evm opp.